pkgs/applications/virtualization/virtualbox/guest-additions-iso/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, lib, virtualbox}:
+
+let
+  inherit (virtualbox) version;
+in
+stdenv.mkDerivation rec {
+  pname = "VirtualBox-GuestAdditions-iso";
+  inherit version;
+
+  src = fetchurl {
+    url = "http://download.virtualbox.org/virtualbox/${version}/VBoxGuestAdditions_${version}.iso";
+    sha256 = "0efbcb9bf4722cb19292ae00eba29587432e918d3b1f70905deb70f7cf78e8ce";
+  };
+
+  dontUnpack = true;
```

Comment:
nit: Probably also `dontBuild` etc?

Reply:
The whole thing can be replaced with `buildCommand`.